ansible/main.yml

```diff
@@ -19,19 +21,25 @@
       when: token is undefined
 
     - name: Create Project
-      command: "{{ oc }} --config=/dev/null new-project {{ project_name }} --display-name='{{ project_display_name }}' --description='{{ project_description }}'"
```

Comment:
@codificat I had errors with the kube config in this section if I didn't use `--config=/dev/null`

Reply:
Steps to reproduce:
1. `ansible-playbook -i inventory.yml main.yml`
2. `oc delete project {dev,stage,prod,registry}`
3. `ansible-playbook -i inventory.yml main.yml`

```
fatal: [stage-1]: FAILED! => {"changed": true, "cmd": ["oc", "--insecure-skip-tls-verify", "--token=io4OtEOkOhgYG0EmgdgBiWr2EwIEzzHL4CkjxnTUr58", "--server=https://10.53.252.73:8443", "-n", "stage", "new-project", "stage", "--display-name=Stage -
 Software lifecycle", "--description=Stage - Software Development Lifecycle (SDLC)"], "delta": "0:00:00.809714", "end": "2017-11-09 12:48:43.717101", "failed": true, "msg": "non-zero return code", "rc": 1, "start": "2017-11-09 12:48:42.907387", "
stderr": "error: yaml: line 13: could not find expected ':'", "stderr_lines": ["error: yaml: line 13: could not find expected ':'"], "stdout": "", "stdout_lines": []}
...ignoring 
```